src/units.cpp

```diff
@@ -358,20 +366,33 @@ void Units::getUnit(size_t index, std::string &reference, std::string &prefix, d
     }
 }
```

Comment:
Minor: the order in the implementation of the different `getUnitAttributes()` methods is not the same as that of their declaration.